sdks/python/apache_beam/ml/transforms/tft.py

```diff
@@ -598,15 +605,34 @@ def __init__(
     self.ngrams_separator = ngrams_separator
     self.name = name
     self.split_string_by_delimiter = split_string_by_delimiter
+    self.compute_word_count = compute_word_count
+    if self.compute_word_count:
```

Comment:
```suggestion
    if compute_word_count:
```

I don't think we actually use this again